Library/Formula/qt.rb

```diff
@@ -25,6 +25,9 @@ class Qt < Formula
     sha256 "c8a0fa819c8012a7cb70e902abb7133fc05235881ce230235d93719c47650c4e"
   end
 
+  # Make `HOMEBREW_PREFIX/lib/qt4/plugins` an additional plugin search path.
+  patch :DATA
```

Comment:
I understand that and it makes sense. The reason I nevertheless went with this approach first is that it was the most straightforward solution with a patch that seemed innocent enough.

Given that Qt 4.8 is EOL by the end of the year (non-commercial support), we should probably ditch it entirely and everything that has a hard dependency on it. (As I assume we're not eager to become the new upstream once the true upstream ditches this release completely.)